src/main.rs

```diff
@@ -120,7 +122,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         )?;
     } else {
         println!("Loading trading post prices");
-        let tp_prices: Vec<api::Price> = request::request_paginated("commerce/prices", &None, notify).await?;
+        print!("Pages:");
+        let commerce_notify = |url: &str| {
+            print!(" {}", &url[51..url.len()-14]);
```

Comment:
What does `url[51..url.len()-14]` extract? Is there a more maintainable way of doing it e.g. by searching for a string?

Reply:
Looks like that's the page number - I made it print just page numbers instead of the whole lines. There's probably a regex extract method - I'll have to look for it when I have some time.